code/infrastructure-api/src/kubeWatcher/kubeWatcher.js

```diff
@@ -7,20 +7,30 @@ import { parsePodLabels } from './kubernetesHelpers';
 import { CREATING, READY } from '../models/stack.model';
 import { STACKS, SELECTOR_LABEL } from '../stacks/Stacks';
 
-const kubeApi = config.get('kubernetesApi');
 const kubeNamespace = config.get('podNamespace');
-const events = ['added', 'modified', 'deleted'];
 const stackNames = Object.values(STACKS).map(stack => stack.name);
+const kc = new k8s.KubeConfig();
 
 function kubeWatcher() {
   logger.info(`Starting kube-watcher, listening for pods labelled "${SELECTOR_LABEL}" on "${kubeNamespace}" namespace.`);
 
-  return new KubeWatch('pods', {
-    url: kubeApi,
-    namespace: kubeNamespace,
-    labelSelector: SELECTOR_LABEL,
-    events,
-  });
+  kc.loadFromDefault();
```

Comment:
This did work locally when using a `yarn start`. I did however try running it in docker-compose after the PR went in and it didn't work so this might need to be revisited.